shared/actions/platform-specific.native.js

```diff
@@ -187,6 +187,12 @@ function configurePush() {
   })
 }
 
+// TODO: persistRouteState() can race with loadRouteState(), i.e. it
```

Comment:
I tried to figure out a good way to do this, but all of them seemed really involved, so I'd rather wait until the next time we hit this bug and see from the logs what happened.

Reply:
i think we can just fix this logically wo this logging. i agree that what we have is a race so we can just do the correct bookeeping to fix this right?